.coveragerc

```diff
@@ -210,6 +210,7 @@ omit =
     homeassistant/components/sensor/gtfs.py
     homeassistant/components/sensor/hp_ilo.py
     homeassistant/components/sensor/imap.py
+    homeassistant/components/sensor/knx.py
```

Comment:
Can you group this sensor with the knx group above? You can migrate the specific platform entries to be 1 that matches all using `*`